rollup/bundleHelpers.js

```diff
@@ -157,7 +157,7 @@ export function makeBundleConfigVariants(baseConfig, options = {}) {
       output: {
         entryFileNames: chunkInfo => `${baseConfig.output.entryFileNames(chunkInfo)}.debug.min.js`,
       },
-      plugins: [terserPlugin],
+      plugins: [includeDebuggingPlugin, terserPlugin],
```

Comment:
This isn't super in scope for this PR (and it was my mistake), so I've fixed it in https://github.com/getsentry/sentry-javascript/pull/5192.